.github/workflows/link-checker.yml

```diff
@@ -1,9 +1,13 @@
 name: Link Checker
+
 on:
   push:
     branches: [ main ]
-  pull_request:
-    branches: [ main ]
+    paths:
+      - 'docs/**'
```

Comment:
do we still need docs/** any longer?

Reply:
Do the below file-types cover all the files within docs?

Reply:
Maybe not, because this tool checks links to images or other files too which docs have

Reply:
images have links?  I don't understand.  This is a directory listing of files that are checked for missing links... 
If you want to keep the docs directory listed here, I don't mind, I just think it's redundant.